### ISSUE TITLE
Connection failure with Sqlite3 error message

### ISSUE THREAD

**Issue by walter-weinmann** [omnisci/omniscidb#668]
### I'm using the official Docker image. The first connection is ok - when I try to connect later I get the following error:

```
2021-07-29 13:52:13,809 [DatabaseSeeder.java] INFO  Start OmniSciDB
java.sql.SQLException: Omnisci connection failed - [OmniSci.java:read:15275:TOmniSciException(error_msg:Sqlite3 Error: disk I/O error)]
        at com.omnisci.jdbc.OmniSciConnection.<init>(OmniSciConnection.java:460)
        at com.omnisci.jdbc.OmniSciDriver.connect(OmniSciDriver.java:80)
        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
        at ch.konnexions.db_seeder.jdbc.AbstractJdbcSeeder.connect(AbstractJdbcSeeder.java:326)
        at ch.konnexions.db_seeder.jdbc.AbstractJdbcSeeder.connect(AbstractJdbcSeeder.java:275)
        at ch.konnexions.db_seeder.jdbc.omnisci.OmnisciSeeder.setupDatabase(OmnisciSeeder.java:93)
        at ch.konnexions.db_seeder.jdbc.AbstractJdbcSeeder.createData(AbstractJdbcSeeder.java:399)
        at ch.konnexions.db_seeder.DatabaseSeeder.main(DatabaseSeeder.java:220)
Caused by: TOmniSciException(error_msg:Sqlite3 Error: disk I/O error)
        at com.omnisci.thrift.server.OmniSci$connect_result$connect_resultStandardScheme.read(OmniSci.java:15275)
        at com.omnisci.thrift.server.OmniSci$connect_result$connect_resultStandardScheme.read(OmniSci.java:15253)
        at com.omnisci.thrift.server.OmniSci$connect_result.read(OmniSci.java:15195)
        at org.apache.thrift.TServiceClient.receiveBase(TServiceClient.java:88)
        at com.omnisci.thrift.server.OmniSci$Client.recv_connect(OmniSci.java:406)
        at com.omnisci.thrift.server.OmniSci$Client.connect(OmniSci.java:391)
        at com.omnisci.jdbc.OmniSciConnection.setSession(OmniSciConnection.java:431)
        at com.omnisci.jdbc.OmniSciConnection.<init>(OmniSciConnection.java:452)
        ... 8 more
Processing of the script was aborted, error code=1
```

**Comment by andrewseidl** [comment]
How are you starting the container, and does the drive holding the `omnisci-storage` directory (if you're passing one in) have at least a few GB of storage available?

Also are you passing in a pre-existing `omnisci-storage` directory or are you letting the container create its own on start?

**Comment by walter-weinmann** [comment]
# Windows 10

```
rmdir /q /s tmp\omnisci-docker-storage
mkdir       tmp\omnisci-docker-storage

docker network create db_seeder_net 2>nul || echo Docker network db_seeder_net already existing
docker create --name    db_seeder_db ^
              --network db_seeder_net ^
              -p        %DB_SEEDER_CONNECTION_PORT%:%DB_SEEDER_CONTAINER_PORT% ^
              -v        %cd%\tmp\omnisci-docker-storage:/omnisci-storage ^
              omnisci/core-os-cpu

echo Docker start db_seeder_db (OmniSciDB %DB_SEEDER_VERSION%) ...
docker start db_seeder_db

ping -n 60 127.0.0.1>nul
```

# Ubuntu 20.04

```
  rm    -rf tmp/omnisci-docker-storage
  mkdir     tmp/omnisci-docker-storage

  docker network create db_seeder_net  2>/dev/null || true
  docker create --name    db_seeder_db \
                --network db_seeder_net \
                -p        "${DB_SEEDER_CONNECTION_PORT}":"${DB_SEEDER_CONTAINER_PORT}"/tcp \
                -v        "$PWD/tmp/omnisci-docker-storage":/omnisci-storage \
                omnisci/core-os-cpu

  echo "Docker start db_seeder_db (OmniSciDB ${DB_SEEDER_VERSION}) ..."
  if ! docker start db_seeder_db; then
      exit 255
  fi

  sleep 60
```

**Comment by walter-weinmann** [comment]
### New error message with DBMS 5.9.0 / JDBC 42.3.1:

```
2021-12-16 13:47:20,798 [DatabaseSeeder.java] INFO  Start OmniSciDB
2021-12-16 13:47:20,802 [AbstractDbmsSeeder.java] INFO  tickerSymbolIntern =omnisci
2021-12-16 13:47:20,808 [AbstractJdbcSeeder.java] INFO  tickerSymbolExtern =omnisci
java.sql.SQLException: Thrift transport connection failed - [TSocket.java:open:243:org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused: connect]
        at com.omnisci.jdbc.OmniSciConnection.<init>(OmniSciConnection.java:456)
        at com.omnisci.jdbc.OmniSciDriver.connect(OmniSciDriver.java:80)
        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
        at ch.konnexions.db_seeder.jdbc.AbstractJdbcSeeder.connect(AbstractJdbcSeeder.java:337)
        at ch.konnexions.db_seeder.jdbc.AbstractJdbcSeeder.connect(AbstractJdbcSeeder.java:286)
        at ch.konnexions.db_seeder.jdbc.omnisci.OmnisciSeeder.setupDatabase(OmnisciSeeder.java:93)
        at ch.konnexions.db_seeder.jdbc.AbstractJdbcSeeder.createData(AbstractJdbcSeeder.java:410)
        at ch.konnexions.db_seeder.DatabaseSeeder.main(DatabaseSeeder.java:208)
Caused by: org.apache.thrift.transport.TTransportException: java.net.ConnectException: Connection refused: connect
        at org.apache.thrift.transport.TSocket.open(TSocket.java:243)
        at com.omnisci.jdbc.OmniSciConnection.manageConnection(OmniSciConnection.java:387)
        at com.omnisci.jdbc.OmniSciConnection.<init>(OmniSciConnection.java:450)
        ... 8 more
Caused by: java.net.ConnectException: Connection refused: connect
        at java.base/sun.nio.ch.Net.connect0(Native Method)
        at java.base/sun.nio.ch.Net.connect(Net.java:579)
        at java.base/sun.nio.ch.Net.connect(Net.java:568)
        at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:588)
        at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
        at java.base/java.net.Socket.connect(Socket.java:633)
        at org.apache.thrift.transport.TSocket.open(TSocket.java:238)
        ... 10 more
Processing of the script was aborted, error code=1
```

**Comment by walter-weinmann** [comment]
### Issue solved with DBMS 5.10.2 / JDBC 5.10.0.
